bus/bus.go

```diff
@@ -437,11 +466,11 @@ func (b *bus) consensusNetworkHandler(jc jape.Context) {
 
 func (b *bus) txpoolFeeHandler(jc jape.Context) {
 	fee := b.tp.RecommendedFee()
-	jc.Encode(fee)
+	b.writeResponse(jc, http.StatusOK, TxPoolResp(fee))
```

Comment:
found a bug where writeResponse is returning this for the txpool recommended fee
```
{
        "Lo": 11553255926290448384,
        "Hi": 1
}
```